tasks/broad/ImputationTasks.wdl

```diff
@@ -884,6 +884,7 @@ task ExtractIDs {
     String bcftools_docker = "us.gcr.io/broad-gotc-prod/imputation-bcf-vcf:1.0.7-1.10.2-0.1.16-1669908889"
     Int cpu = 1
     Int memory_mb = 4000
+    Boolean for_dependency
```

Comment:
is this going to break the other (minimac) Imputation.wdl that uses this task?

Reply:
yea i'll set a default value